bridge/checkpoint/checkpoint.go

```diff
@@ -0,0 +1,138 @@
+package checkpoint
+
+import (
+	"github.com/0xPolygon/polygon-edge/bridge/checkpoint/transport"
+	"github.com/0xPolygon/polygon-edge/bridge/sam"
+	"github.com/0xPolygon/polygon-edge/network"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+)
+
+const (
+	EpochSize = 100
```

Comment:
What value we want to have for checkpoint EpochSize?
1. Some default value
2. Same as epoch size for consensus
3. Cli config to setup + default value

Reply:
@Kourin1996 Ping

Reply:
As discussed, must be No.3 and Epoch size can be given from CLI config or genesis.json

Reply:
Agree, @lazartravica you said we should go for the same value (consensus epochSize = checkpoint epochSize), but I agree on this with Yoshiki. What do you think?

Reply:
@Aleksao998 We can use same value but default IBFT epoch size is too big

Reply:
That's why I agree with you that we should have separate flags.